cjworkbench/tests/i18n/test_trans.py

```diff
@@ -33,20 +52,24 @@ def test_format_invalid_default(self):
             "Hello you ! 2",
         )
 
-    # Tests that a translator can't break our system by including a numeric variable in the message
-    def test_format_invalid_message(self):
-        self.assertTrue(
+    # Tests that a translator can use a numeric variable in the message
```

Comment:
Please test that an extra numeric variable in `message` doesn't break our system.